src/textual/demo.py

```diff
@@ -141,7 +141,7 @@
 - **DataTable** A spreadsheet-like widget for navigating data. Cells may contain text or Rich renderables.
 - **TreeControl** An generic tree with expandable nodes.
 - **DirectoryTree** A tree of file and folders.
-- *... [many more planned](https://textual.textualize.io/roadmap/) ...* 
+- *... many more planned ...* 
```

Comment:
Awww! I liked the links. What was the issue?

Reply:
For some bizarre reason iTerm gets slow when there are links on the page. Just one link makes it stuttery.

Reply:
Ugh! That's a shame! But, yeah, doesn't make sense to slow the demo for this. :-/